### DISCUSSION TITLE
Enabling SpellCheck in HtmlContentEditor in Serenity

### DISCUSSION THREAD

**Discussion by DrastiGohel** [serenity-is/Serenity#6386] [Help]
I want to enable browser spelling check into HtmlContentEditor, is it possible to achieve it? Please let me know.

![image](https://user-images.githubusercontent.com/98172041/166066603-78e3f875-08b8-4836-9690-608502ab3106.png)

**Comment by wirble** [comment]
Unless you need bold and underline, why not just use the regular TextAreaEditor?  If you need to use html, it seems like it's using CKEditor 4.  You may need to hack Serenity.Assets or maybe add in another plugins mentioned in the pages below:

https://github.com/serenity-is/Serenity/blob/master/src/Serenity.Assets/wwwroot/Scripts/ckeditor/README.md
https://ckeditor.com/docs/ckeditor4/latest/features/spellcheck.html
